docs/EN_US/ECLLanguageReference/ECLR_mods/BltInFunc-BUILD.xml

```diff
@@ -894,6 +894,28 @@ BUILD(FilterDsLib1);
             compression on the payload. The resulting index can be smaller
             than using lz4.</entry>
           </row>
+
+          <row>
+            <entry><emphasis role="bold"><emphasis
+            role="bold">'inplace:lz4s'</emphasis> </emphasis></entry>
+
+            <entry>Causes inplace compression on the key fields and lz4s
+            compression on the payload. This uses the streaming API to build
```

Comment:
This uses the stream LZ4 API to avoid recompressing the data and reduce the index build times.

Similar for lz4shc below.